cmd/airflow.go

```diff
@@ -162,6 +168,50 @@ func newAirflowInitCmd() *cobra.Command {
 	return cmd
 }
 
+func newAirflowUpgradeTestCmd(astroClient astro.Client) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "upgrade-test",
+		Short:   "Run some tests to see if your environment and DAGs are compatable with a new version of Airflow or Runtime",
```

Comment:
```suggestion
		Short:   "Run tests to see if your environment and DAGs are compatable with a new version of Airflow or Runtime",
```

nit: some sounds vague, maybe Jake would be the right person to take a look at it